ConcordiumNetSdk.sln

```diff
@@ -7,19 +7,23 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Concordium.Sdk", "src\Conco
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Concordium.Sdk.Tests.UnitTests", "tests\Tests\UnitTests\Concordium.Sdk.Tests.UnitTests.csproj", "{FA99FA9D-C722-4BD4-BB6E-C83843EC5BA0}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Concordium.Sdk.Examples.RawClient.GetAccountInfo", "examples\Examples\RawClient\GetAccountInfo\Concordium.Sdk.Examples.RawClient.GetAccountInfo.csproj", "{B5E9C218-21CD-42E9-BB37-1D7AB3BAB2C6}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "examples", "examples", "{D86A7796-6AD5-48CA-9DC6-415CC53629B6}"
```

Comment:
I refactored the example structure due to
- It has folder debt not needed
- When opening your solution it looks more clean to have it in a nested structure, like it is in the file system. Without it’s difficult to see what is SDK and what is example - see example

<img width="385" alt="Screenshot 2023-05-16 at 11 47 40" src="https://github.com/Concordium/concordium-net-sdk/assets/132270889/80e7342a-3a08-4d96-a1e8-0347c3cb541b">

- I shortened the name since I felt it was redundant to have “Example” in names and also the SDK name

Reply:
Change back since the value changing it is this PR is to small.